FireRender.Maya.Src/Context/TahoeContext.cpp

```diff
@@ -570,84 +564,61 @@ bool TahoeContext::IsAOVSupported(int aov) const
 	return (aov != RPR_AOV_VIEW_SHADING_NORMAL) && (aov != RPR_AOV_COLOR_RIGHT);
 }
 
-bool TahoeContext::IsPhysicalLightTypeSupported(PLType lightType) const
+bool NorthStarContext::IsPhysicalLightTypeSupported(PLType lightType) const
 {
-	if (lightType == PLTDisk || lightType == PLTSphere)
-	{
-		return m_PluginVersion == TahoePluginVersion::RPR2;
-	}
+	assert(m_CoreVersion == RPRCoreVersion::RPR2);
 
 	return true;
 }
 
-bool TahoeContext::IsGLInteropEnabled() const
-{
-	return m_PluginVersion == TahoePluginVersion::RPR1;
-}
-
-bool TahoeContext::MetalContextAvailable() const
+bool NorthStarContext::MetalContextAvailable() const
 {
     return true;
 }
 
-bool TahoeContext::IsDeformationMotionBlurEnabled() const
+bool NorthStarContext::IsDeformationMotionBlurEnabled() const
 {
-	return TahoeContext::IsGivenContextRPR2(this);
+	assert(NorthStarContext::IsGivenContextRPR2(this));
```

Comment:
should we remove this function "IsGivenContextRPR2"? since its always Northstar here from now on